src/01-functions.js

```diff
@@ -9,7 +9,9 @@
  *  alwaysTrue();
  *  //> true
  */
-function alwaysTrue() {}
+function alwaysTrue() {
+  return true;q
```

Comment:
let watch extra characters like this 'q` before we commit.  Remember no code will get executed after the return statement so while this will still run we don't want to push this code.